src/components/Text/CustomText.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Ref, forwardRef } from "react";
+import { assignInlineVars } from "@vanilla-extract/dynamic";
+import Text, { TextBaseProps } from "./Text";
+import { Combine } from "@/types/utils";
+import { TypoScale, TypoType, theme } from "@/styles/theme.css";
+
+type CustomTextBaseProps = {
+  as?: TypoType;
+  size?: TypoScale;
+  children: React.ReactNode;
+};
+
+export type CustomTextProps = Combine<CustomTextBaseProps, TextBaseProps>;
+
+function CustomText(
+  {
+    as = "body",
+    size = "md",
+    display,
+    color,
+    lineHeight,
+    weight,
+    align,
+    family,
+    children,
+    ...props
+  }: CustomTextProps,
+  ref: Ref<any>
+) {
+  const fontSizeValue = theme.fontSize[as][size];
+
+  const dynamicStyles = assignInlineVars({
```

Comment:
```suggestion
  const dynamicStyles = assignInlineVars<Record<string, string>>({
```

https://github.com/vanilla-extract-css/vanilla-extract/blob/master/packages/dynamic/src/assignInlineVars.ts

구현보니까 요렇게 꽂아도 들어갈 것 같은데, 혹시 안되나유?? 👀

Reply:
해결했습니다🥲